setup/webpack.config.js

```diff
@@ -2,6 +2,8 @@ module.exports = async () => {
     require('../src/index');
     require(Mix.paths.mix());
 
+    await Mix.dispatch('internal:gather-dependencies');
+    await Mix.dispatch('internal:install-dependencies');
```

Comment:
I named these this way because we might want to refactor this implementation later and I would not want extensions to rely on these events as they're an implementation detail.